lib/ansible/modules/cloud/amazon/ec2_instance.py

```diff
@@ -1515,12 +1565,17 @@ def main():
 
         module.params['filters'] = filters
 
+    if module.params.get('cpu_options'):
```

Comment:
This can be simplified as `if module.params.get('cpu_options') and not module.botocore_at_least('1.10.16'):`

Reply:
That's updated thanks!